src-tauri/src/northstar/launch_arguments.rs

```diff
@@ -0,0 +1,34 @@
+#[tauri::command]
+pub fn get_launch_arguments(game_path: &str) -> Result<Vec<String>, ()> {
+    // If file does not exist, we return an empty list
+    let launch_args_path = format!("{}/ns_startup_args.txt", game_path);
+    if !std::path::Path::new(&launch_args_path).exists() {
+        return Ok(vec![]);
+    }
+
+    let data = match std::fs::read_to_string(launch_args_path.clone()) {
+        Ok(content) => content,
+        Err(_) => {
+            return Ok(vec![]);
+        }
+    };
+
+    Ok(data.split_whitespace().map(|arg| arg.to_string()).collect())
+}
+
+#[tauri::command]
+pub fn set_launch_arguments(game_path: &str, arguments: Vec<String>) -> Result<(), String> {
+    // If file does not exist, we return an empty list
+    let launch_args_path = format!("{}/ns_startup_args.txt", game_path);
+    if !std::path::Path::new(&launch_args_path).exists() {
+        return Err("Launch arguments file does not exist.".to_string());
+    }
```

Comment:
I think we should just create it if it doesn't exist yet ^^

Reply:
Addressed in 25d32c0fe1f05ec3766bf4c433efa56d8b04488d